code/style.css

```diff
@@ -1,7 +1,267 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+body {
+  margin: 0;
+}
+
+main, header, section, aside, footer {
+  margin: 0;
+  border: none;
+  color: #5B5B5B ;
+  font-family: Helvetica, Arial, sans-serif;
+  font-size: small;
+}
+
+main {
+  background: #FFFFFF;
+}
+
+.header {
+  background: #AAD6EC;
+  padding: 20px;
+  color: #FFFFFF;
+  display: flex;
+  justify-content: center;
+}
+
+.hero {
+  background: #5B5B5B ;
+  color: #FFFFFF ;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  /* position: relative; */
+}
+
+.hero-image {
+  width: 100vw;
+  height: 100wh;
+  object-fit: cover;
+  -webkit-filter: grayscale(100%); 
+} 
+
+.hero-image:hover { 
+-webkit-filter: grayscale(0%); 
+-webkit-transition: -webkit-filter 1s;
+}
+
+.mobile-hero {
+  width: auto;
```

Comment:
In order to make img responsive on mobile version ,   width: 100%; can be implemented instead of auto